### ISSUE TITLE
Numbered Bookmarks - Modified file Notification-Dialog may be Excessive (and not necessary)

### ISSUE THREAD

**Issue by neoh4x0r** [geany/geany-plugins#1218]
I've noticed that the accept/reject notification dialog is displayed for files which have been modified outside of geany (which may or may not have been changed), at least I don't remember having made changes, outside of geany, every time it is displayed. 

I find my self just clicking **ignore** (which seems to be often) and just moving on.

I'm wondering if:
1. Is the notification dialog really necessary? (eg. could it just force the rejection and log a status message)
2. or maybe the display of the notification dialog could be a plugin configuration option (if it is disabled -- then maybe it could be auto-rejected like in the first part and perhaps that could be configured as well)

Long story short -- I really don't think displaying a modal notification, when a file is loaded, is really necessary when all someone wants to do (is ignore/reject) and load the standard bookmarks (which from my experience aren't saved by geany).

----

This following code blob in **geanynumberedbookmarks.c** is responsible for showing the dialog:

https://github.com/geany/geany-plugins/blob/ddf06e8a1ce945a9d3dbdf482dc9aeb8478f080c/geanynumberedbookmarks/src/geanynumberedbookmarks.c#L967-L981

**Comment by elextr** [comment]
> I'm wondering if: The notification dialog is really necessary? (eg. could it just force the rejection and log a status message)

Well, only you know if its an error with the timestamp, or something "touch"ed the file without changing it, or you edited the file with __ANOTHER__ editor (gasps of horror at such infidelity ;-), the plugin can't tell, so it asks you.

IIRC the discussion at the time saving bookmarks and folds was contentious because if the file was changed they won't be in the same place.  So the modification check was used to reduce that risk.  

Better that you find why you get timestamp mismatches.  If something touches the file just don't do that.  If its an error in the timestamp there are several possible causes.  Depending on the OS and the filesystem the timestamp may not be updated immediately after the file is saved, so the value the plugin reads is the previous time it was saved.  This is particularly likely for remote filesystems.
